src/main/java/com/cx/restclient/CxSASTClient.java

```diff
@@ -624,8 +678,9 @@ private void uploadZipFile(byte[] zipFile, long projectId) throws CxClientExcept
             MultipartEntityBuilder builder = MultipartEntityBuilder.create();
             builder.setMode(HttpMultipartMode.BROWSER_COMPATIBLE);
             builder.addPart(ZIPPED_SOURCE, streamBody);
+            String apiVersion = getContentTypeAndApiVersion(config, SCAN_WITH_SETTINGS_URL);
```

Comment:
The function should have been called for the API SAST_ZIP_ATTACHMENTS instead of SCAN_WITH_SETTINGS_URL